Meta/CMake/utils.cmake

```diff
@@ -57,7 +57,9 @@ function(serenity_libc target_name fs_name)
         target_link_libraries(${target_name} clang_rt.builtins)
         # FIXME: Implement -static-libstdc++ in the next toolchain update.
         target_link_options(${target_name} PRIVATE -nostdlib++ -Wl,-Bstatic -lc++ -Wl,-Bdynamic)
-        target_link_options(${target_name} PRIVATE -Wl,--no-dependent-libraries)
+	if (NOT USE_MOLD_LINKER)
+            target_link_options(${target_name} PRIVATE -Wl,--no-dependent-libraries)
+	endif()
```

Comment:
kinda weird spaces here. I think we want this indented to be clearly under the Clang$ block?

Reply:
Accidentally had a tab there instead of 4 spaces. 😬